core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/stack/upscale/StackUpscaleActions.java

```diff
@@ -102,9 +106,10 @@
     @Inject
     private EnvironmentClientService environmentClientService;
 
-    @Bean(name = "UPSCALE_PREVALIDATION_STATE")
-    public Action<?, ?> prevalidate() {
+    @Bean(name = "UPDATE_DOMAIN_DNS_RESOLVER_STATE")
```

Comment:
what happens if UPSCALE_PREVALIDATION_STATE was the last save flow log and CB is upgraded? There will be no UPSCALE_PREVALIDATION_STATE, so I think flow will fail. Am I right?

Reply:
I see, the next step is this :) sorry.

Reply:
yes, new state was placed before the prevalidation state